flexmeasures/utils/entity_address_utils.py

```diff
@@ -236,14 +349,14 @@ def build_ea_scheme_and_naming_authority(
                 [domain_parts.subdomain, domain_parts.domain, domain_parts.suffix],
             )
         )
-        if config_var_domain_key in current_app.config.get(
+        if domain_parts.domain in ("localhost", "127.0.0.1"):
```

Comment:
I moved this further up so tests running against localhost can use this function.